src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html

```diff
@@ -12,6 +12,7 @@
         <img class="first-photo" [src]="firstFriend" alt="First friend photo">
         <img class="second-photo" [src]="secondFriend" alt="Second friend photo">
       </div>
+      <div class="edit" (click)="goEditHabit()"></div>
```

Comment:
what is goEditHabit ?

Reply:
changed

Reply:
This is function to navigate to habit profile